scripts/build/TestPlatform.targets

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TestPlatformRoot>$(MSBuildThisFileDirectory)..\..\</TestPlatformRoot>
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
-    <NoWarn>$(NoWarn);CA1416;RS0037</NoWarn>
+    <NoWarn>$(NoWarn);CA1416;RS0037;CS1570;CS1572;CS1573;CS1574;CS1584;CS1591;CS1658;CS1712;CS1734</NoWarn>
```

Comment:
The cleanup of these warning suppressions (which were all ignored due to `GenerateDocumentationFile` set to `false`) is left for a separate PR.

Reply:
Could you please create a ticket for these TODOs and also add code comment about what are these warnings.

Reply:
Sure. I've opened #4236